src/show/components/TextField/index.js

```diff
@@ -44,11 +51,18 @@ const TextField = ({
 }) => {
   const [error, setError] = useState('');
   const [inputType, setInputType] = useState(type);
+  const errorColor = value.trim() === '' ? 'text-red-500' : 'text-blue-500';
```

Comment:
what will happen if do this 

`const errorColor = `text-${value.trim() === '' ? 'red' : 'blue'}-500`;
`

Reply:
this will do same like When we type something in the input field, the text and border color will turn blue, and when we remove it, the text n border color will turn red

Reply:
try to use template literal as much as you can